trade-finance-manager-ui/server/errors/invalid-environment-variable.error.js

```diff
@@ -1,8 +1,3 @@
-class InvalidEnvironmentVariableError extends Error {
-  constructor(message) {
-    super(message);
-    this.name = this.constructor.name;
-  }
-}
+const { InvalidEnvironmentVariableError } = require('@ukef/dtfs2-common');
 
 module.exports = InvalidEnvironmentVariableError;
```

Comment:
I'm slightly hesitant to bloat this PR too much with refactoring this - if you think worthwhile I'll go for it

Reply:
I think this might be slightly messy to keep as is, as it establishes an antipattern for future work :(. Once everything else is approved could we quickly do it please and ill just check that commit and well be good to go. Also note dtfs-central-api\external-api\gef-ui has it too. Should hopefully just be as easy as find replacing imports